docs/using-keystone-webhook-authenticator-and-authorizer.md

```diff
@@ -109,6 +108,104 @@ $ kubectl apply -f /etc/kubernetes/keystone-auth/policy-config.yaml
 As you can see, the version 2 policy definition is much simpler and
 more succinct.
 
+For many scenarios clients require access to `nonresourse` paths.
+`nonresource` paths include: `/api`, `/apis`, `/metrics`, `/resetMetrics`,
+`/logs`, `/debug`, `/healthz`, `/swagger-ui/`, `/swaggerapi/`, `/ui`, and
+`/version`. Clients require access to `/api`, `/api/*`, `/apis`, `/apis/*`,
+and /version to discover what resources and versions are present on the
+server. Access to other `nonresource` paths can be disallowed without
+restricting access to the REST api.
```

Comment:
> Access to other `nonresource` paths can be disallowed without restricting access to the REST api.

I'm a little confused about this line, what do you want to describe?

Reply:
This is just kind of continuation of the previous sentence. Basically I just meant that access to `/api`, `/api/*`, `/apis`, `/apis/*`and `/version` is essential and access to /metrics`, `/resetMetrics`, `/logs`, `/debug`, `/healthz`, `/swagger-ui/`, `/swaggerapi/`, `/ui` is optional.